samples/client/petstore-security-test/cpp-qt5/client/OAIFakeApi.h

```diff
@@ -0,0 +1,52 @@
+/**
+ * OpenAPI Petstore *_/ ' \" =end -- \\r\\n \\n \\r
+ * This spec is mainly for testing Petstore server and contains fake endpoints, models. Please do not use this for any other purpose. Special characters: \" \\  *_/ ' \" =end --       
+ *
+ * OpenAPI spec version: 1.0.0 *_/ ' \" =end -- \\r\\n \\n \\r
+ * Contact: something@something.abc *_/ ' \" =end -- \\r\\n \\n \\r
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+#ifndef _OAI_OAIFakeApi_H_
+#define _OAI_OAIFakeApi_H_
+
+#include "OAIHttpRequest.h"
+
+#include "OAIObject.h"
+#include "OAIUNKNOWN_BASE_TYPE.h"
```

Comment:
Is that supposed to be like that?

Reply:
This code is generated in the latest master branch as well (I could reproduce that). The change is not the result of the modifications in my pull request. Please see also my comment [here](https://github.com/OpenAPITools/openapi-generator/pull/1366#discussion_r230567682). It seems that the samples for the c++-qt5 based code generator were not updated on a regular basis. I updated the samples only because I touched the 'AbstractCppCodegen'.